content/id/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -0,0 +1,234 @@
+- **DNS**: Buat [headless service](/docs/concepts/services-networking/service/#headless-services) dengan pod selektor yang sama,
+  dan temukan DaemonSet menggunakan _resource_ `endpoints` atau mengambil beberapa A _record_ dari DNS.
+- **Service**: Buat Servis dengan pod selektor yang sama, dan gunakan Servis untuk mengakses daemon pada
+  node random. (Tidak ada cara mengakses spesifik node)
+
+## Melakukan Pembaruan DaemonSet
+
+Jika label node berubah, DaemonSet akan menambahkan Pod ke node cocok yang baru dan menghapus Pod dari
+node tidak cocok yang baru.
+
+Anda bisa mengubah Pod yang dibuat DaemonSet. Namun, Pod tidak membolehkan perubahan semua _field_.
+Perlu diingat, _controller_ DaemonSet akan menggunakan templat yang asli di waktu selanjutnya 
+node baru (bahkan dengan nama yang sama) dibuat.
+
+Anda bisa menghapus DaemonSet. Jika Anda spesifikasikan `--cascade=false` dengan `kubectl`, maka
+Pod akan dibiarkan pada node. Jika Anda pada waktu kemudian membuat DaemonSet baru dengan selektor
+yang sama, DaemonSet yang baru akan mengadopsi Pod yang sudah ada. Jika ada Pod yang perlu diganti,
+DaemonSet akan mengganti sesuai dengan `updateStrategy`.
+
+Anda bisa [melakukan rolling update](/docs/tasks/manage-daemon/update-daemon-set/) pada DaemonSet.
+
+## Alternatif DaemonSet
+
+### _Init Scripts_
+
+Anda mungkin menjalankan proses _daemon_ dengan cara menjalankan mereka langsung pada node (e.g. 
+menggunakan `init`, `upstartd`, atau `systemd`). Tidak ada salahnya seperti itu. Namun, ada beberapa
+keuntungan menjalankan proses _daemon_ via DaemonSet.
+
+- Kemampuan memantau dan mengatur log _daemon_ dengan cara yang sama dengan aplikasi.
+- Bahasa dan alat Konfigurasi yang sama (e.g. Templat pod, `kubectl`) untuk _daemon_ dan aplikasi.
+- Menjalankan _daemon_ dalam kontainer dengan batasan _resource_ meningkatkan isolasi antar _daemon_ dari
+  kontainer aplikasi. Namun, hal ini juga bisa didapat dengan menjalankan _daemon_ dalam kontainer tapi
+  tanpa Pod (e.g. dijalankan langsung via Docker).
+
+### Pod Polosan
+
+Dimungkinkan untuk membuat Pod langsung dengan menspesifikasikan node mana untuk dijalankan. Namun,
+DaemonSet akan menggantikan Pod yang untuk suatu alasan dihapus atau dihentikan, seperti pada saat
+kerusakan node atau pemeliharaan node yang mengganggu seperti pembaruan _kernel_. Oleh karena itu, Anda
+perlu menggunakan DaemonSet daripada membuat Pod satu per satu.
+
+### Pod Statis
+
+Dimungkinkan untu membuat Pod dengan menulis sebuah berkas ke direktori tertentu yang di-_watch_ oleh Kubelet.
+Pod ini disebut dengan istilah [pod statis](/docs/concepts/cluster-administration/static-pod/).
+Berbeda dengan DaemonSet, Pod statis tidak dapat dikelola menggunakan kubectl atau klien API Kubernetes
+yang lain. Pod statis tidak bergantung kepada apiserver, membuat pod statis berguna pada kasus-kasus
+_bootstrapping_ klaster.
+
+
+### Deployments
```

Comment:
Hello
I have some feedback here that I hope is useful.
```suggestion
### Deployment
```
because that's an English plural not a bahasa Indonesia plural.
Alternatively: use bahasa Indonesia pluralization style. I don't speak the language myself but I think that could be, with duplication:
```suggestion
### Deployment-deployment
```
?

Reply:
+1

Reply:
I'll go with singular `Deployment`, because it looks less cluttered and the pluralization is not really emphasized in this context.